src/main/python/NetworkTableManager.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2023 FRC 5990 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+# associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions: The above copyright notice and this
+# permission notice shall be included in all copies or substantial portions of the Software. THE SOFTWARE IS PROVIDED "AS IS",
+# WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# If this library is not installed, don't install ntcore but pyntcore.
+import ntcore
+import sys
+import time
+
+CONNECTION_TIMEOUT_SECONDS = 30
+CONNECTION_COOLDOWN_SECONDS = 0.1
+
+
+def terminate_client(network_table_instance: ntcore.NetworkTableInstance, client_name: str):
+    print("Terminating client named {}".format(client_name))
+    ntcore.NetworkTableInstance.destroy(network_table_instance)
+
+
+def passed_connection_timeout(starting_time):
```

Comment:
in this context, you should use `past_connection_timeout` instead of `passed_connection_timeout`.
see this for more info: https://www.reddit.com/r/grammar/comments/9v5y3x/is_it_im_past_this_or_is_it_im_passed_this_if/